shrinker-test/src/main/java/com/example/NoSerializedNameMain.java

```diff
@@ -19,37 +20,40 @@ static class TestClassNotAbstract {
     public String s;
   }
 
-  static class TestClassWithoutDefaultConstructor {
+  static class TestClassHasArgsConstructor {
     public String s;
 
     // Specify explicit constructor with args to remove implicit no-args default constructor
-    public TestClassWithoutDefaultConstructor(String s) {
+    public TestClassHasArgsConstructor(String s) {
       this.s = s;
     }
   }
 
   /**
-   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_DefaultConstructor()}.
+   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_NoArgsConstructor()}.
    */
-  public static String runTest() {
-    TestClass deserialized = new Gson().fromJson("{\"s\":\"value\"}", same(TestClass.class));
+  public static String runTestNoArgsConstructor() {
+    TestClassNoArgsConstructor deserialized = new Gson().fromJson(
+        "{\"s\":\"value\"}", same(TestClassNoArgsConstructor.class));
     return deserialized.s;
   }
 
   /**
-   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_DefaultConstructorNoJdkUnsafe()}.
+   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_NoArgsConstructorNoJdkUnsafe()}.
    */
   public static String runTestNoJdkUnsafe() {
     Gson gson = new GsonBuilder().disableJdkUnsafe().create();
-    TestClassNotAbstract deserialized = gson.fromJson("{\"s\": \"value\"}", same(TestClassNotAbstract.class));
+    TestClassNotAbstract deserialized = gson.fromJson(
+        "{\"s\": \"value\"}", same(TestClassNotAbstract.class));
     return deserialized.s;
   }
 
   /**
-   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_NoDefaultConstructor()}.
+   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_ConstructorHasArgs()}.
```

Comment:
The method name seems to be incorrect
```suggestion
   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_HasArgsConstructor()}.
```

Reply:
Fixed, thanks.